config/initializers/assets.rb

```diff
@@ -26,6 +26,4 @@
 Rails.application.config.assets.precompile += %w( potlatch2/FontLibrary.swf )
 Rails.application.config.assets.precompile += %w( potlatch2/locales/*.swf )
 Rails.application.config.assets.precompile += %w( help/introduction.* )
-Rails.application.config.assets.precompile += %w( iD/img/*.svg iD/img/*.png iD/img/*.gif )
-Rails.application.config.assets.precompile += %w( iD/img/pattern/*.png )
-Rails.application.config.assets.precompile += %w( iD/locales/*.json )
```

Comment:
There is a README and some example code that we can remove, but other than that I think most of the files are needed.

We want to get to the point we only fetch the sign definitions for the country that a user is looking at.  As of now we're prefetching the json files for the handful of places where Mapillary supports street sign detection.. But that's the reason there are so many json files.